includes/admin/upgrades/upgrade-functions.php

```diff
@@ -392,14 +396,6 @@ function give_v132_upgrade_give_payment_customer_id() {
 	/* @var Give_Updates $give_updates */
 	$give_updates = Give_Updates::get_instance();
 
-	if ( ! current_user_can( 'manage_give_settings' ) ) {
```

Comment:
@DevinWalker Remove user permission check from update callback function. Sometimes cause stalled process if we set empty `give_completed_upgrades` manually.

Reply:
👍